src/components/ProfilePicture.tsx

```diff
@@ -0,0 +1,52 @@
+import { Box, BoxProps, Skeleton, SxProps } from '@mui/material';
+import { useLocation } from 'react-router-dom';
+import { StyledBaseContributorIndicator } from '../pages/registration/contributors_tab/ContributorIndicator';
+import { dataTestId } from '../utils/dataTestIds';
+import { useProfilePicture } from '../utils/hooks/useProfilePicture';
+import { getContributorInitials } from '../utils/registration-helpers';
+
+interface ProfilePictureProps extends Pick<BoxProps, 'sx'> {
+  personId: string;
+  fullName: string;
+  height: string;
+  hasBorder?: boolean;
+  sx?: SxProps;
+}
+
+export const ProfilePicture = ({ personId, fullName, height, hasBorder, sx }: ProfilePictureProps) => {
+  const { profilePictureQuery, profilePictureString } = useProfilePicture(personId);
+  const isPublicPage = useLocation().pathname.includes('research-profile');
```

Comment:
I tilfelle vi endrer path en dag
```suggestion
  const isPublicPage = useLocation().pathname.includes(UrlPathTemplate.ResearchProfile);
```

Reply:
Kanskje rename til `isResearchProfile` også? Tolket det litt som at det var alle offentlige visninger først, men ser nå at det bare skal være akkurat denne siden.